ember/app/ui/cases/new/controller.js

```diff
@@ -53,6 +61,19 @@ export default class CaseNewController extends Controller.extend(
       mutation: createCaseMutation,
       variables: { form: this.selectedForm, workflow: workflow.id },
     });
+
+    const user = yield this.store.queryRecord("identity", {});
+
+    yield this.apollo.mutate({
+      mutation: saveWorkItem,
+      variables: {
+        input: {
+          workItem: newCase.saveCase.case.workItems.edges[0].node.id,
+          assignedUsers: [user.id],
```

Comment:
This should be done automatically through the `addressed_groups` right?

Reply:
Removed as it is obsolete